js/index.js

```diff
@@ -1,9 +1,31 @@
-let books = [];
+/* eslint-disable max-classes-per-file */
+class BooksDatabase {
+  constructor() {
+    this.booksList = [];
+  }
+
+  addNewBook(bookInfo) {
+    this.booksList.push(bookInfo);
+  }
+
+  deleteBook(bookInfo) {
+    this.booksList = this.booksList.filter((element) => element.bookTitle !== bookInfo.bookTitle);
+  }
+}
+
+const Books = new BooksDatabase();
 
 function searchID(id) {
   return document.getElementById(id);
 }
 
+class Book {
+  constructor(title, author) {
+    this.bookTitle = title;
+    this.bookAuthor = author;
+  }
+}
```

Comment:
- [OPTIONAL] This suggetion is optional if you choose not to go with the suggestion in the above review comment. But if you choose to filter books by ID, which I highly recommend, then your Book class will need to accept an `id` field as suggested below. This `id` field will be autogenerated for every book object.
```suggestion
class Book {
  constructor(title, author) {
    this.id = !Books.booksList.length ? 1 : Books.booksList[Books.booksList.length - 1].id + 1;
    this.bookTitle = title;
    this.bookAuthor = author;
  }
}
```